encoder/vaapiencpicture.h

```diff
@@ -124,6 +124,8 @@ createMiscObject(VAEncMiscParameterType miscType, T * &bufPtr)
     if (obj) {
         misc->type = miscType;
         bufPtr = (T *) (misc->data);
+        if (bufPtr)
```

Comment:
This if is not needed, data is an array in misc, if we can access misc->type, misc->data never be NULL.